boards/BMS/Inc/App/App_PreChargeSequence.h

```diff
@@ -1,39 +1,82 @@
 #pragma once
 
 #include <stdbool.h>
+#include <stdint.h>
 
-struct PreChargeSequence;
+struct PreCharge;
 
 /**
  * Allocate and initialize a pre-charge sequence
- * @param enable_pre_charge_sequence A function that can be called to enable
+ * @param enable A function that can be called to enable
  * the pre-charge sequence
- * @param disable_pre_charge_sequence A function that can be called to disable
+ * @param disable A function that can be called to disable
  * the pre-charge sequence
+ * @param get_ts_voltage A function that can be called to get the tractive
+ * system voltage in V
+ * @param min_duration_ms The time associated with the
+ * inverter/charger capacitors being charged to 90% of the accumulator voltage
  * @return The created pre-charge sequence, whose ownership is given to the
```

Comment:
Missing a couple param descriptions here

Reply:
Don't see anything missing atm

Reply:
I was referring to these:
```
    uint32_t min_duration_ms,
    uint32_t min_complete_duration_ms,
    uint32_t max_complete_duration_ms
```

Reply:
Fixed this

Reply:
Fixed now